app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorUpdate.kt

```diff
@@ -33,6 +33,18 @@ class SyncIndicatorUpdate : Update<SyncIndicatorModel, SyncIndicatorEvent, SyncI
       is DataForSyncIndicatorStateFetched -> updateSyncIndicatorState(model, event.currentTime, event.syncIndicatorFailureThreshold)
       is IncrementTimerTick -> incrementTimer(model, event)
       is DataSyncErrorReceived -> handleDataSyncErrors(event.errorType)
+      SyncIndicatorViewClicked -> syncIndicatorClicked(model.lastSyncedState)
+    }
+  }
+
+  private fun syncIndicatorClicked(lastSyncedState: LastSyncedState?): Next<SyncIndicatorModel, SyncIndicatorEffect> {
+    val isSyncing = lastSyncedState != null && lastSyncedState.lastSyncProgress != SYNCING
```

Comment:
I am a little confused why the property is named `isSyncing` and in the condition we check `lastSyncedState.lastSyncProgress != SYNCING`. Is this intentional?

Reply:
Looking at the usage, I am assuming it should be named `isNotSyncingCurrectly`. Correct if I am wrong.

Reply:
Yes you are right. I will change the condition instead.